english/avatar/air.md

```diff
@@ -10,5 +10,5 @@ The next step is other planets.
 
 You feel stranded.
 How can I get off of this planet?
-You are desperate and scream, "Oh God!"
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.